torchdr/affinity/entropic.py

```diff
@@ -907,3 +907,80 @@ def _compute_log_affinity(self, X: torch.Tensor):
             log_affinity_matrix -= math.log(n_samples_in)
 
         return log_affinity_matrix
+
+
+class NormalizedStudentAffinity(LogAffinity):
+    r"""
+    Computes the Student affinity matrix :math:`(1 + \mathbf{C} / \sigma)^{-1}`
```

Comment:
Cosmit: docstrings should follow a format where the first line is a simple sentence fitting on one line (often starting with a verb) ending with a dot, followed by a blank line. 
Here: 
```
"""
Compute the Student affinity matrix.

The formula is given by ...
"""
```

See for example https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/ensemble/_base.py#L139